crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/utils/helpers.py

```
@@ -0,0 +1,16 @@
+"""Utility module for testing regular import aliases."""
+
+
+def helper_function():
```

Ruff (0.11.9)

Missing return type annotation for public function `helper_function`

Add return type annotation: `str`

(ANN201)